doc/develop/toolchains/arm_compiler.rst

```diff
@@ -0,0 +1,52 @@
+.. _toolchain_armclang:
+
+Arm Compiler 6
+##############
+
+#. Download and install a development suite containing the `Arm Compiler 6`_
+   for your operating system.
+
+#. :ref:`Set these environment variables <env_vars>`:
+
+   - Set :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` to ``armclang``.
+   - Set :envvar:`ARMCLANG_TOOLCHAIN_PATH` to the toolchain installation
+     directory.
+
+#. The Arm Compiler 6 needs the :envvar:`ARMLMD_LICENSE_FILE` environment
+   variable to point to your license file or server.
+
+For example:
+
+   .. code-block:: console
+
+      # Linux, macOS, license file:
+      export ARMLMD_LICENSE_FILE=/<path>/license_armds.dat
+      # Linux, macOS, license server:
+      export ARMLMD_LICENSE_FILE=8224@myserver
+
+      # Windows, license file:
+      > set ARMLMD_LICENSE_FILE=c:\<path>\license_armds.dat
+      # Windows, license server:
+      > set ARMLMD_LICENSE_FILE=8224@myserver
```

Comment:
better use another code-clock using the batch lexer, https://pygments.org/docs/lexers/#pygments.lexers.shell.BatchLexer

Reply:
fixed